microprofile/lra/client/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2021 Oracle and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.microprofile.lra</groupId>
+        <artifactId>helidon-microprofile-lra-project</artifactId>
+        <version>2.3.1-SNAPSHOT</version>
+    </parent>
+
+    <groupId>io.helidon.microprofile.lra</groupId>
+    <artifactId>helidon-microprofile-lra-client-parent</artifactId>
+    <name>Helidon MicroProfile LRA Client Parent</name>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>spi</module>
+        <module>core</module>
```

Comment:
`core` module is not named according to guidelines - in this case, it should just be called `client`